docs/compute-element/hosted-ce.md

```diff
@@ -0,0 +1,139 @@
+OSG Hosted CE
+=============
+
+
+Introduction
+------------
+
+An OSG Compute Element (CE) is the entry point for jobs coming from the OSG;
+it handles authorization and delegation of jobs to your existing campus HPC/HTC cluster.
+
+Many sites set up their compute element locally.
+As an alternative, OSG offers a hosted CE option
+wherein the OSG team will host and operate the HTCondor compute element,
+and configure it for the science communities that you choose to support.
+
+This document explains the requirements and the procuedure for obtaining a hosted CE.
+If you wish to have a local compute element instead,
+see page for [installing HTCondor-CE](/compute-element/install-htcondor-ce).
+
+![managed services diagram](/compute-element/img/managed_services_diagram.png)
+
+
+Before Starting
+---------------
+
+Before preparing your cluster for OSG jobs, consider the following requirements:
+
+-   An existing compute cluster with a [supported batch system](/index.md#prepare-the-batch-system)
+    running on a [supported operating system](/release/supported_platforms)
+
+-   Outbound network connectivity from the compute nodes (they can be behind NAT)
+
+-   A Unix account on your cluster's submit server, accessible via an SSH key
+    The OSG CE will use this account to automatically submit jobs,
+    so it must also have permissions to submit jobs to the batch system
+
+-   If your batch system is not HTCondor,
+    there must be a shared file system between the submit server and the compute nodes
```

Comment:
Let's add a `See this section for details` with a link to the section you added below

Reply:
updated

Reply:
updated